themes/bootstrap3/js/common.js

```diff
@@ -7,6 +7,10 @@ window.console = window.console || { log: function polyfillLog() {} };
 var VuFind = (function VuFind() {
   var defaultSearchBackend = null;
   var path = null;
+  var lightbox = {
```

Comment:
I just realized that this is most likely not the place where this should be added. I will change this.